src/ui/utils/getTypeSelectOptions.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import type {SelectOption} from '@gravity-ui/uikit';
+import type {DATASET_FIELD_TYPES, WithRequired} from 'shared';
+import DataTypeIcon from 'ui/components/DataTypeIcon/DataTypeIcon';
+
+import {getDatasetLabelValue} from './helpers';
+
+export const getTypeSelectOptions = (types: DATASET_FIELD_TYPES[]): SelectOption[] => {
```

Comment:
Why this helper is located out of dataset unit?

Reply:
Because it used in https://github.com/datalens-tech/datalens-ui/blob/befddc7911dd1c6231f55402a3489006e9698954/src/ui/components/DialogParameter/helpers.tsx#L24